### ISSUE TITLE
How to fine-tune the existing CNN model?

### ISSUE THREAD

**Issue by sunyinhui** [kevinlin311tw/caffe-cvprw15#6]
I‘m very curious about how you do it. Which codes are modified or written ???   you add a new layer the model is change, how to fine tune the  alexnet.caffemodel??? 
Thanks a lot !!!

**Comment by kevinlin311tw** [comment]
1) The latent layer mentioned in the CVPRW15 paper is actually the fully connected layer with 48 or 128 neurons. We embedded only two additional layers (fully connected layer + sigmoid layer) between fc7 and fc8. You may check our train_val.prototxt [here (latent layer)](https://github.com/kevinlin311tw/caffe-cvprw15/blob/master/examples/cvprw15-cifar10/train_CIFAR10_48.prototxt#L344), and [here (sigmoid)](https://github.com/kevinlin311tw/caffe-cvprw15/blob/master/examples/cvprw15-cifar10/train_CIFAR10_48.prototxt#L365).

2) About the fine-tuning process, we follow the standard fine-tuning method. You may check [our training script](https://github.com/kevinlin311tw/caffe-cvprw15/blob/master/examples/cvprw15-cifar10/train_48.sh), or [caffe's example](http://caffe.berkeleyvision.org/gathered/examples/finetune_flickr_style.html). The pre-trained weights will be loaded into the new model, matching layers by name. Then, we apply Gaussian random initialization for the weights of the latent layer. Finally, we applied SGD to fine-tune the entire network, that is, to optimize the weights and biases of every layer.

**Comment by sunyinhui** [comment]
Thank you !!!  >^_^<
